<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <widget name="__qt_fake_top_level">
  <widget class="QLineEdit" name="EditClinicEmailInput">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ObstericsandGynecologyCheckBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Obstetrics and Gynecology</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="UrologyCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Urology</string>
   </property>
  </widget>
  <widget class="QLabel" name="ClinicContactNumberLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Number:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PediatricsCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pediatrics</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveDetailsButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Details</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditAddress2Input">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditClinicNameInput">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="NeurologyCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neurology</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="OrthopedicSurgeryCheckBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Orthopedic Surgery</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RadiologyCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radiology</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CardiologyCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cardiolody</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address2Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address Line 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditAddress1Input">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditContactNumberInput">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="PsychiatryCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Psychiatry</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="GeneralPracticeCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>General Practice</string>
   </property>
  </widget>
  <widget class="QLabel" name="EditClinicDetailsButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit Clinic Details</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="EndocrinologyCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Endocrinology</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address1Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address Line 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ClinicEmailLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="DermatologyCheckBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dermatology</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="SurgeryCheckBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Surgery (General)</string>
   </property>
  </widget>
  <widget class="QLabel" name="ClinicNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clinic Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SpecialityLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Speciality:</string>
   </property>
  </widget>
 </widget>
 <resources/>
</ui>
